core/benches/kura.rs

```diff
@@ -61,7 +61,7 @@ async fn measure_block_size_for_n_validators(n_validators: u32) {
     block_store.write(&block).await.unwrap();
     let metadata = fs::metadata(
         block_store
-            .get_block_path(NonZeroU64::new(1_u64).unwrap())
+            .get_block_path(NonZeroU64::new(2_u64).unwrap()) // TODO: Figure out why this fails with 1_u64
```

Comment:
I guess that's because `BlockStore::get_block_path` is supposed to be used in `BlockStore::write` to write a new block.
`1` equals to the max of the existing filenames as the base heights so it fails due to [this line](https://github.com/hyperledger/iroha/blob/b2a353ca9f0dbfc618a722d4f94b2c78a6607bbd/core/src/kura.rs#L337)